cards.js

```diff
@@ -1,15 +1,18 @@
 var Createbutton = document.getElementById("create");
-var Deletebutton = document.getElementById("delete");
 
 Createbutton.addEventListener("click", createCard);
 
 function createCard() {
 	var textElement = document.getElementById("userInput");
-	//console.log("verify textElement", textElement.value);
-	// var textCard = textElement.value;
+
 	 var textCard = document.getElementById("output");
-	 textCard.innerHTML = "<div>" + textElement.value + "</div>";
+	 textCard.innerHTML = "<div>" + textElement.value + "</div><button id='delete'>Delete</button>";
```

Comment:
It looks like you have the delete button on the text box rather than the individual cards.